baseband/vlbi_base/base.py

```diff
@@ -108,9 +118,33 @@ def _get_time(self, header):
 
     @lazyproperty
     def time0(self):
-        """Start time."""
+        """Start time of file."""
         return self._get_time(self.header0)
 
+    @property
+    def header0(self):
+        """First header of file."""
+        return self._header0
+
+    @property
+    def bps(self):
+        """Bits per sample."""
+        return self._bps
+
+    @property
+    def complex_data(self):
+        """If `True`, decoded data is complex."""
```

Comment:
I don't particular like the `If True` - maybe just `Whether the decoded data is complex.`?